### PR TITLE
The whole implementation

### PR DESCRIPTION
**Pull request by MohammadShahin**
All of the solution is implemented here.

### CODE REVIEW

**Comment by ilov3** [comment]
@MohammadShahin I've noticed an issue in your `output.csv` file in `gpu_model` column: it is required to be like for example `NVIDIA GeForce GT 1030`, not just `NVIDIA GeForce GT`

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
 # Web Scraping with Provectus
```
Comment (ilov3) [inline README.md:1]: Could you please separate task1 readme into separate readme file and move it into `/task1` folder?

File: task1/gpu_scraper.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                    if key == self.gpu_model_key:
+                        value_list = [word for word in value.split() if word.isascii()]
+                        value = ' '.join(value_list)
+                        break
+                except:
+                    continue
+            return value
+        except Exception as e:
+            print("Error while finding the GPU's model:", e)
+            return ''
```
Comment (ilov3) [inline task1/gpu_scraper.py:95]: There should be easier way to achieve this

File: task1/gpu_scraper.py [hunk 3]
```diff
@@ -0,0 +1,219 @@
+import time
+from typing import Tuple
+
+from bs4 import BeautifulSoup
+from task1.utils import read_link, get_fixed_text
+import pandas as pd
```
Comment (ilov3) [inline task1/gpu_scraper.py:6]: `pandas` is a bit of overkill for this task and requires another bunch of dependencies, could you please implement this task without it?

**Comment by ilov3** [comment]
Overall looks good enough!

File: task1/gpu_scraper.py (hunk 2 above)
Comment (MohammadShahin) [reply task1/gpu_scraper.py:95]: I could not find a better way, to be honest. It would be great if you could show how to do that properly.

Comment (ilov3) [reply task1/gpu_scraper.py:94]: As a tip - I would use here `xpath` selector with `contains` [function](https://developer.mozilla.org/en-US/docs/Web/XPath/Functions/contains). Also in which cases you need this filtering - `value_list = [word for word in value.split() if word.isascii()]`?

**Comment by ilov3** [comment]
Could you please reply with "Fixed/Done" on comments which you've already resolved it will help understand that it's fixed and could be checked. Thank you!

File: README.md (hunk 1 above)
Comment (MohammadShahin) [reply README.md]: Done

File: task1/gpu_scraper.py (hunk 2 above)
Comment (MohammadShahin) [reply task1/gpu_scraper.py:94]: Some names contained words like "Видеокарта", which I needed to exclude from the name.

File: task1/gpu_scraper.py (hunk 3 above)
Comment (MohammadShahin) [reply task1/gpu_scraper.py]: Done

**Pull request merged**